cmd/bootstrap/main.go

```diff
@@ -78,6 +78,10 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		zap.S().Error("Failed to initialize telemetry batcher: ", err)
 	}
 
+	if err := process.LogAndReportVersion(ctx); err != nil {
```

Comment:
why not have this as part of process.Exec? then we get it automatically everywhere

Reply:
Started to do it but changed because:
1) It seemed to magical. We already have a problem with people trying to debug what's happening, I think a little repeating code in different binaries for clarity is a good thing. 
2) If a satellite running 1.0.1 has a critical bug and we want to patch and fix, we release a new version 1.0.2, if we remove 1.0.1 as an acceptable version and add 1.0.2, we force every binary, to update to a new binary that doesn't impact them at all. If that ever happens we will then need to split out the config and I think having it explicit in each binary is better.

Reply:
haha ok I see you made the same conclusion about breaking versions up